docs/assets/scripts/hedwig-main.umd.js

```diff
@@ -7118,12 +7119,10 @@
 
 
     connectedCallback() {
-      this.defaults = {};
-      var defaults = new Defaults();
-      this.defaults.margin = this.dataset.margin || defaults.margin;
-      this.defaults.height = (this.dataset.height || defaults.graphHeight) - this.defaults.margin.top - this.defaults.margin.bottom;
-      this.defaults.width = (this.dataset.width || defaults.graphWidth) - this.defaults.margin.left - this.defaults.margin.right;
-      this.defaults.lineColor = this.dataset.lineColor || defaults.lineColor;
+      this.defaults.margin = this.dataset.margin || this.defaults.margin;
+      this.defaults.height = (this.dataset.height || this.defaults.height) - this.defaults.margin.top - this.defaults.margin.bottom;
```

Comment:
this.dataset.height was not working if only we provide it as property binding, not as an input variable.

Hope it working now.